package.json

```diff
@@ -51,6 +51,9 @@
     "_php-cs-fixer": "docker-compose run --rm server-php dev-tools/vendor/bin/php-cs-fixer fix --config=php-cs-fixer.config.php"
   },
   "dependencies": {},
+  "resolutions": {
+    "jquery": "3.4.1"
+  },
```

Comment:
Setting `"jquery": "3.4.x"` in `client` didn't work for me (jquery 3.5.0 got installed anyhow, I guess a subdependency uses it and I ended with webpack errors). This fixed the issue for me...